### PR TITLE
ui2: Themed Header component

### PR DESCRIPTION
**Pull request by amanalili**
DO NOT MERGE, branched out from art/themedFont

#### PR Requirements

If you have made **any** visual changes to the GUI. Make sure you have:
- [ ] Tested on iOS Tablet
- [ ] Tested on small Android

### CODE REVIEW

File: src/components/navigation/HeaderWalletSelector.js [hunk 1]
```diff
@@ -34,50 +34,49 @@ class HeaderWalletSelectorComponent extends React.Component<Props> {
...
-  (state: RootState): StateProps => {
-    const walletId = state.ui.wallets.selectedWalletId
-    const selectedWallet = state.ui.wallets.byId[walletId]
-    return {
-      selectedWalletName: selectedWallet ? selectedWallet.name : null,
-      selectedWalletCurrencyCode: state.ui.wallets.selectedCurrencyCode
-    }
-  },
+  (state: RootState): StateProps => ({
+    selectedWalletName: UI_SELECTORS.getSelectedWallet(state) ? UI_SELECTORS.getSelectedWallet(state).name : null,
```
Comment (paullinator) [inline src/components/navigation/HeaderWalletSelector.js]: don’t use simple selectors. they are being deprecated

Comment (paullinator) [reply src/components/navigation/HeaderWalletSelector.js]: just access the state directly

Comment (paullinator) [reply src/components/navigation/HeaderWalletSelector.js]: like the old code did

File: src/components/navigation/HeaderWalletSelector.js [hunk 2]
```diff
@@ -14,6 +14,10 @@ import type { Dispatch, RootState } from '../../types/reduxTypes.js'
 import { ArrowDownTextIconButton } from '../common/ArrowDownTextIconButton.js'
 import { Airship } from '../services/AirshipInstance.js'
 
+type OwnProps = {
+  isNameOnly?: boolean
```
Comment (paullinator) [inline src/components/navigation/HeaderWalletSelector.js]: change to showNameOnly

File: src/components/Main.ui.js [hunk 3]
```diff
@@ -194,7 +194,6 @@ export class MainComponent extends React.Component<Props> {
                 component={ifLoggedIn(CreateWalletChoiceComponent)}
                 renderTitle={<HeaderTitle title={s.strings.title_create_wallet} />}
                 renderLeftButton={this.renderBackButton(s.strings.title_wallets)}
-                renderRightButton={this.renderEmptyButton()}
```
Comment (paullinator) [inline src/components/Main.ui.js:201]: wrong commit title. you are removing the empty button not back button

**Review by paullinator — CHANGES_REQUESTED** [review]

File: src/components/navigation/HeaderWalletSelector.js (hunk 2 above)
Comment (amanalili) [reply src/components/navigation/HeaderWalletSelector.js]: changed to showWalletNameOnly

**Comment by amanalili** [comment]
Fixed comments

**Review by paullinator — APPROVED** [review]

**Pull request closed without merging**